actions/class.Runner.php

```diff
@@ -1100,7 +1100,7 @@ private function setNavigationContextToCommand(NavigationContextAwareInterface $
     {
         $command->setNavigationContext(
             $this->getRequestParameter('direction') ?? '',
-            $this->getRequestParameter('scope'),
+            $this->getRequestParameter('scope') ?? '',
```

Comment:
Do we want to make `NavigationContextAwareTrait::setNavigationContext(string $direction, string $scope, ?string $ref): void` to allow `$scope` to be null  instead? That is, changing the signature in the trait instead of fixing the method call (I think that would need changes in the method signatures for getters in `NavigationContextAwareTrait` as well)

Reply:
**thought:** I think both proposals are good. Actually, the direction scope is not mandatory. In the current issue being fixed, the direction is `exit`, and the `scope` is useless since we are exiting from the test. Having it scoped to `item` or any other element would make no sense. With this respect, the solution of having a nullable scope in the signature would be better, of course, as it would prevent a similar issue to occur elsewhere or later. But the current fix looks good enough too.

Reply:
Then I think we can make every parameter nullable, right? And turn them into empty strings inside the trait: 6e57b0ea3f08bbe8ea7ed37b395966ba08020f52

Reply:
As we are using [nullable parameters](https://www.php.net/manual/en/migration71.new-features.php) I'm adding a require for PHP 7.1 in `composer.json`  (btw other places  such as `ItemContextAwareTrait` were already using them too) :point_right: 5075331589e0e70cca39b6de9e3ecb805977732d